config/environments/production.rb

```diff
@@ -23,7 +23,7 @@
   # Disable serving static files from the `/public` folder by default since
   # Apache or NGINX already handles this.
   config.serve_static_files = ENV['RAILS_SERVE_STATIC_FILES'].present?
-  config.serve_static_assets = true
```

Comment:
we've gotten deprecation warnings about this line. Apparently line 25 should handle it - I double checked that there is a heroku config variable by that name, but if we see problems, we can just explicitly change that to be true.